syntax = "proto3";

package proto_lat_lon_in_city;

message LatLonInCity
{
  bool lat_lon_in_city   = 1;
}